prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs

```diff
@@ -17939,111 +17949,6 @@ the Province.
-                            Email = "",
-                            Name = "Medinet"
-                        },
-                        new
-                        {
-                            Code = 25,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Plexia"
-                        },
-                        new
-                        {
-                            Code = 26,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "PharmaClik"
-                        },
-                        new
-                        {
-                            Code = 27,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Nexxsys"
-                        },
-                        new
-                        {
-                            Code = 28,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Kroll"
-                        },
-                        new
-                        {
-                            Code = 29,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Assyst Rx-A"
-                        },
-                        new
-                        {
-                            Code = 30,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "WinRx"
-                        },
-                        new
-                        {
-                            Code = 31,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Shoppers Drug Mart HealthWatch NG"
-                        },
-                        new
-                        {
-                            Code = 32,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Commander Group"
-                        },
-                        new
-                        {
-                            Code = 33,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "BDM"
-                        },
-                        new
-                        {
-                            Code = 34,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Meditech"
-                        },
-                        new
-                        {
-                            Code = 35,
-                            CareSettingCode = 1,
-                            Email = "",
-                            Name = "Cerner"
```

Comment:
All of these removed lines should be in the snapshot. I'm not sure how you local got into this state, but could you please regenerate your migration? After removing the migration you may have to do `git checkout develop -- C:\Path\to\the\repo\prime-dotnet-webapi\Migrations\ApiDbContextModelSnapshot.cs` to fetch the snapshot that's currently on develop.